packages/rocketchat-setup-wizard/client/setupWizard.js

```diff
@@ -0,0 +1,268 @@
+import s from 'underscore.string';
+
+const setSettingsAndGo = (settings, registerServer = true) => {
+	const settingsFilter = Object.entries(settings)
+		.filter(key => !/registration-|registerServer|currentStep/.test(key))
+		.map(setting => {
+			return {
+				_id: setting[0],
+				value: setting[1]
+			};
+		});
+
+	settingsFilter.push({
+		_id: 'Statistics_reporting',
+		value: registerServer
+	});
+
+	RocketChat.settings.batchSet(settingsFilter, function(err) {
+		if (err) {
+			return handleError(err);
+		}
+
+		localStorage.setItem('wizardFinal', true);
+		FlowRouter.go('setup-wizard-final');
+	});
+};
+
+Template.setupWizard.onCreated(function() {
+	if (localStorage.getItem('wizardFinal')) {
+		FlowRouter.go('setup-wizard-final');
+	}
+
+	this.hasAdmin = new ReactiveVar(false);
+	this.state = new ReactiveDict();
+	this.wizardSettings = new ReactiveVar([]);
+	this.invalidEmail = new ReactiveVar(false);
+
+	const storage = JSON.parse(localStorage.getItem('wizard'));
+	if (storage) {
+		Object.entries(storage).forEach(([key, value]) => {
+			this.state.set(key, value);
+		});
+	}
+
+	Meteor.call('serverHasAdminUser', (error, result) => {
```

Comment:
Can't you check the `Meteor.user().roles.includes('admin')` instead of calling the server for that?

Reply:
Because I need to check if the server has an admin user, not if my user (that I don't have if is the first access to the server) is admin.

Reply:
Why do you need to check if there is no admin in client side? Wouldn't be better to control that setting that enable or disable the wizard?